### ISSUE TITLE
Typo: "RtFuncion" instead of "RtFunction"

### ISSUE THREAD

**Issue by ghost** [reinforced/Reinforced.Typings#160]
Asp per title.

```
public class RtFuncion : RtMember, IDecoratable
    {
        //
        // Summary:
        //     Constructs new instance of AST node
        public RtFuncion();

        //
        // Summary:
        //     Function name
        public RtIdentifier Identifier { get; set; }
        //
        // Summary:
        //     Function return type
        public RtTypeName ReturnType { get; set; }
        //
        // Summary:
        //     Function parameters
        public List<RtArgument> Arguments { get; }
        //
        public List<RtDecorator> Decorators { get; }
        //
        // Summary:
        //     Function body (supplied as raw text)
        public RtRaw Body { get; set; }
        //
        public override IEnumerable<RtNode> Children { get; }

        //
        public override void Accept(IRtVisitor visitor);
        //
        public override void Accept<T>(IRtVisitor<T> visitor);
    }
}
```

**Comment by nros** [comment]
seems to be fixed with commit c35f249c4a3a439dd559ef302db8ac9815463f0b

**Comment by pavel-b-novikov** [comment]
Community force! :D Combo!
